test/unit/EmptyState.spec.js

```diff
@@ -0,0 +1,55 @@
+import { mount, createLocalVue } from '@vue/test-utils';
+import BootstrapVue from 'bootstrap-vue';
+import EmptyState from '../../src/components/EmptyState.vue';
+
+const localVue = createLocalVue();
+localVue.use(BootstrapVue);
+
+test('Component is mounted properly', () => {
```

Comment:
Use this test to make a snapshot test in order to see if the component in mounted properly, and create another test to put the expectation "expect(wrapper.findAll('.add-button-container').length).toBe(1);"